pkgdb2/templates/package.html

```diff
@@ -68,7 +68,6 @@
         <td colspan="2" align="center">
           <div id="monitoring_block">
             <input type="radio" id="no_monitoring" name="monitoring" value="0" />
-            {{ package.monitoring_status }}
```

Comment:
This doesn't seem related to this change.
Was this a mistake, or another change you just wanted to sneak in at the same time?
(which is fine. Just want to be sure it's intended)